src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/EmptyForIteratorPadCheck.java

```diff
@@ -51,6 +51,19 @@
  * &lt;module name=&quot;EmptyForIteratorPad&quot;/&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * for (Iterator it = map.entrySet().iterator();  it.hasNext(); ); // violation since whitespace
+ *                                                                 //after semicolon
+ *
+ * for (Iterator foo = very.long.line.iterator();
+ *       foo.hasNext();
+ *      ); // ok
+ *
+ * for (Iterator i = m.entrySet().iterator();  i.hasNext();); // ok
```

Comment:
example block should be (it should be rendered like this in HTML) to make it ease to understand what this Check care about:

```
for (Iterator it = map.entrySet().iterator();  i.hasNext(););   // ok
for (Iterator it = map.entrySet().iterator();  it.hasNext(); ); // violation since whitespace
                                                                //after semicolon

for (Iterator foo = very.long.line.iterator();
     foo.hasNext();
     ); // ok

```

Reply:
Please copy code that I provided to you in your patch.
All names matters.

Reply:
@romani done